tests/unittests/CoreGraphics/CGContextTests.mm

```diff
@@ -262,3 +272,146 @@ void cgContextPathCompare(NSArray* expected, NSArray* result) {
     CGContextRelease(context);
     CGColorSpaceRelease(rgbColorSpace);
 }
+
+bool operator==(const CGPoint& lhs, const CGPoint& rhs) {
+    return ((std::abs(lhs.x - rhs.x) < 0.00001) && (std::abs(lhs.y - rhs.y) < 0.00001));
+}
+
+template <typename T>
+std::basic_ostream<T>& operator<<(std::basic_ostream<T>& os, const CGPoint& pt) {
+    return os << "{" << pt.x << ", " << pt.y << "}";
+}
+
+template <typename T>
+std::basic_ostream<T>& operator<<(std::basic_ostream<T>& os, const CGAffineTransform& transform) {
+    os << "[";
+    if (CGAffineTransformIsIdentity(transform)) {
+        os << "identity transform";
+    } else {
+        os << transform.a << " " << transform.b << " " << transform.c << " " << transform.d << " " << transform.tx << " " << transform.ty;
+    }
+    return os << "]";
+}
+
+class ContextCoordinateTest
```

Comment:
You may want to keep this under win32 target for test portability for compat testing.